tools/release/changelog-renderer.js

```diff
@@ -0,0 +1,26 @@
+const { default: defaultChangelogRenderer } = require('nx/changelog-renderer');
```

Comment:
[Question] Is there any reason to not put the scripts from `tools/` into `packages/repo-tools/src` instead? Having two folders with tools is a little confusing IMO. I'd personally rather put everything into this internal package.

This `const { default: ... }` pattern to me in particular is something I'd really like to avoid if possible. Even if it means having to build the `repo-tools` package before running certain commands.

Reply:
Yes there is currently - npm publish hard errors when it is triggered from within a nested “npm workspace” which is what that package is . I had it there first because I forgot about that npm publish behaviour, I moved it up to tools out of necessity.

Having said that I seem to have the opposite intuition to you - having things which help manage the whole workspace live in a nested project is confusing to me /shrug

Reply:
Heh yeah now that you mention it... it is kind of weird to put non-package things in a package... and `tools/` is less overhead than a full package. For me 90% of happiness here comes from unifying. So if we go to just everything in `tools/` that works for me too!

Reply:
Cool I'll revisit the thread we had on it in discord and do a proper reconciliation in a follow up